aronnax/core.py

```diff
@@ -218,9 +218,9 @@ def interpret_requested_data(requested_data, shape, config):
             with fortran_file(requested_data, 'r') as f:
                 return f.read_reals(dtype=np.float64)
     else:
-        if shape == "3d":
-            return depths(grid, *requested_data)
-        if shape == "2dx":
-            return wind_x(grid, requested_data)
+        if shape == "3dT":
```

Comment:
General comment on programming style: It would have been more stylish to include the change to the `shape ==` tests in the previous commit, rather than this one.  Then that becomes a semantically complete change in its own right that doesn't break any running code; whereas this way, the code is broken after that commit and before this one.  No need to do anything about this now, though.